spec/integration/lib/grape/middleware/logger_spec.rb

```diff
@@ -15,6 +15,7 @@
     expect(subject.logger).to receive(:info).with ''
     expect(subject.logger).to receive(:info).with %Q(Started POST "/api/1.0/users" at #{subject.start_time})
     expect(subject.logger).to receive(:info).with %Q(Processing by TestAPI/users)
+    expect(subject.logger).to receive(:info).with %Q(     Headers: {})
```

Comment:
Yeah, checkout the factory here: https://github.com/ridiculous/grape-middleware-logger/blob/master/spec/factories.rb#L31